config/scout.php

```diff
@@ -100,6 +100,20 @@
 
     'identify' => env('SCOUT_IDENTIFY', false),
 
+    /*
+    |--------------------------------------------------------------------------
+    | Scout Jobs
+    |--------------------------------------------------------------------------
+    |
+    | @TODO add description
+    |
+    */
+
+    'jobs' => [
+        'make_searchable' => Laravel\Scout\Jobs\MakeSearchable::class,
+        'remove_from_search' => Laravel\Scout\Jobs\RemoveFromSearch::class,
+    ],
```

Comment:
Please avoid using the config file for this. All our first party libraries use a static property on a library specific class. We'd need to create a new one for Scout like: https://github.com/laravel/cashier-stripe/blob/master/src/Cashier.php#L69